capture.sh

```diff
@@ -11,9 +20,28 @@ else
   exit 1
 fi
 
+PERIODI=4 #default to 4 seconds between captures. Can be changed with --period command line option
+if [ ! -z "${period}" ]
+    then
+      PERIODI="$period"
+fi
+
+if [ ! -z "${app}" ] && test "$(uname)" != 'Darwin'
+    then
+      echo 'The --app option is currently only available on macOS.'
+      exit 0
```

Comment:
Zero means success, what do you think about returning a non-zero value if `--app` is set for non supported OSes?

Reply:
Changed it!